jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -307,6 +333,12 @@ protected HiveConnection(String uri, Properties info,
     sessConfMap = connParams.getSessionVars();
     setupLoginTimeout();
     if (isKerberosAuthMode()) {
+      // Ensure UserGroupInformation includes any authorized Kerberos principals.
+      LOG.debug("Configuring Kerberos mode");
+      Configuration config = new Configuration();
+      config.set("hadoop.security.authentication", "Kerberos");
```

Comment:
use `AuthTypes.KERBEROS.getAuthName`

Reply:
changed constant.
I don't thing it should  always be kerberized.